fixtures/tests/access_snv/files_metadata.json

```diff
@@ -8,28 +8,7 @@
             "modified_date": "2020-01-06T19:59:50.583Z",
             "file": "fffff52a-0696-48d4-9ca5-6580daa44ab3",
             "version": 0,
-            "metadata": {
```

Comment:
I deleted this metadata to reflect the fact that it won't be present after the bam generation pipeline finishes